scss/events-map.scss

```diff
@@ -330,8 +330,8 @@ section#header {
     }
   }
   div#spiel {
-    padding-top: 10px;
-    margin-left: 10px;
+    padding-top: 15px;
+    margin-left: 15px;
```

Comment:
The map homepage already has these styles applied? So the code here is being overriden or not used? I don't understand why this is defined twice in the css.

Reply:
Whenever these styles were last edited, the .css was edited and not the SCSS. I merely changed the scss to match the generated css.